js/lykke2.js

```diff
@@ -0,0 +1,946 @@
+                'fetchFundingHistory': false,
+                'fetchFundingRate': false,
+                'fetchFundingRateHistory': false,
+                'fetchFundingRates': false,
+                'fetchIndexOHLCV': true,
+                'fetchMarkets': true,
+                'fetchMarkOHLCV': false,
+                'fetchMyTrades': true,
+                'fetchOHLCV': 'emulated',
+                'fetchOpenOrders': true,
+                'fetchOrder': true,
+                'fetchOrderBook': true,
+                'fetchOrders': true,
+                'fetchOrderTrades': true,
+                'fetchPositions': false,
+                'fetchPremiumIndexOHLCV': false,
+                'fetchTicker': true,
+                'fetchTickers': true,
+                'fetchTime': false,
+                'fetchTrades': true,
+                'fetchTradingFees': false,
+                'fetchWithdrawals': true,
+                'setLeverage': false,
+                'setMarginMode': false,
+                'withdraw': true,
+            },
+            'timeframes': {
+                '1m': 'Minute',
+                '5m': 'Min5',
+                '15m': 'Min15',
+                '30m': 'Min30',
+                '1h': 'Hour',
+                '4h': 'Hour4',
+                '6h': 'Hour6',
+                '12h': 'Hour12',
+                '1d': 'Day',
+                '1w': 'Week',
+                '1M': 'Month',
+            },
+            'requiredCredentials': {
+                'apiKey': true,
+                'secret': false,
+            },
+            'urls': {
+                'logo': 'https://lykke-marketing-website.cdn.prismic.io/lykke-marketing-website/7dd44480-31ea-45c8-a246-c97989cd9da6_logo.svg',
+                'api': {
+                    'public': 'https://hft-apiv2.lykke.com/api',
+                    'private': 'https://hft-apiv2.lykke.com/api',
+                },
+                'www': 'https://www.lykke.com',
+                'doc': [
+                    'https://hft-apiv2.lykke.com/swagger/ui/index.html',
+                    'https://lykkecity.github.io/Trading-API',
+                ],
+                'fees': 'https://support.lykke.com/hc/en-us/articles/115002141125-What-are-the-fees-and-charges-', // zero fee
+            },
+            'api': {
+                'public': {
+                    'get': [
+                        'assetpairs',
+                        'assetpairs/{id}',
+                        'assets',
+                        'assets/{id}',
+                        'isalive',
+                        'orderbooks',
+                        'tickers',
+                        'prices',
+                        'trades/public/{AssetPairId}',
+                    ],
+                },
+                'private': {
+                    'get': [
+                        'balance',
+                        'trades',
+                        'trades/order/{OrderId}',
```

Comment:
you need to define the ratelimits here as it is shown in https://lykkecity.github.io/Trading-API/#request-rate-limits